thunder/benchmarks/targets.py

```diff
@@ -166,13 +167,18 @@ def thunder_fwd(b: Benchmark, compile_fn: Callable):
 
         @wraps(cfn)
         def wrapper(*args):
-            return cfn(args)
+            result = cfn(args)
+            if os.getenv("LIGHTNING_BENCH_TRACE") is not None:
```

Comment:
Is the problem we're trying to solve that people would like to see the trace but don't know where to insert a breakpoint?

Reply:
Yep! Or, well, "insert a dump" more like; I doubt there's much pdb use for this situation. Generally when an executor's underlying library causes a failure, the developer sees "test_foo" failed, and the devs' first question is, "well, what is Thunder sending to my executor?"

Reply:
I see. To make it universally available for all tests and interactive scripts I think the best place to put this dump would be here:
https://github.com/Lightning-AI/lightning-thunder/blob/9107a3d8e9bb32345ddc5bf1df5883bc8e4ac458/thunder/__init__.py#L595

Reply:
Yeah, that's a *much* better solution! Will update it to this when I can